spark/v3.3/spark/src/main/java/org/apache/iceberg/spark/source/SparkCopyOnWriteScan.java

```diff
@@ -123,8 +127,17 @@ public void filter(Filter[] filters) {
               tasks().stream()
                   .filter(file -> fileLocations.contains(file.file().path().toString()))
                   .collect(Collectors.toList());
+
+          LOG.info(
+              "{}/{} tasks for table {} matched runtime file filter",
+              filteredTasks.size(),
+              tasks().size(),
+              table().name());
+
           resetTasks(filteredTasks);
         }
+      } else {
+        LOG.warn("Unsupported runtime filter {}", filter);
```

Comment:
should we include concerned table name as well in log

Reply:
Well, I wasn't sure how useful it will be in this context. The table name does not impact whether a filter can be converted, so I matched what we have in other places that convert filters.